03-postgres-typeorm/README.md

```diff
@@ -0,0 +1,96 @@
+# NestJS Boilerplate
+
+This is a powerful NestJS boilerplate for building scalable and enterprise-level applications. It includes robust features such as proper logging mechanisms, JWT-based authentication, OpenAPI documentation, and PostgreSQL setup with TypeORM. You can use this boilerplate as a solid foundation for projects that need to scale to millions of users.
+
+## Getting Started
+
+Follow these simple steps to get your project up and running:
+
+### Prerequisites
+
+Before you begin, ensure you have met the following requirements:
+
+- Node.js installed (v18+ recommended)
+- PostgreSQL with Pgadmin installed and running
+- Git installed
+- A code editor of your choice (e.g., Visual Studio Code)
+
+### Installation
+
+1. Clone the repository:
+
+```bash
+git clone 
+
+```
+
+2. Change to the project directory:
+
+```bash
+cd nestjs-boilerplates/
+
+```
+
+3. Update values of `.env` file according to your configuration needs. The `.env` file is used to store environment-specific variables.
+
+4. Install project dependencies:
+
+```bash
+npm install
+
+```
+
+### Development
+
+To start the development server, run the following command:
+
+```bash
+npm run start:dev
+
+```
+
+This will launch your NestJS application in development mode.
+
+### Production
```

Comment:
Do you have this functionality to separate the env files for prod and dev, if not then please remove this